tests/testsuite/cargo_env_config.rs

```diff
@@ -192,10 +192,15 @@ fn env_applied_to_target_info_discovery_rustc() {
             "src/main.rs",
             r#"
             fn main() {
-                let mut args = std::env::args().skip(1);
-                let env_test = std::env::var("ENV_TEST").unwrap();
-                eprintln!("WRAPPER ENV_TEST:{env_test}");
-                let status = std::process::Command::new(&args.next().unwrap())
+                let mut cmd = std::env::args().skip(1).collect::<Vec<_>>();
+                if cmd.get(1).map(|s| &**s) == Some("-vV") {
+                    // This is the version query, not the target info query, so skip this.
```

Comment:
I guess this raises the question, should the env stuff also be set for the `-vV` query?

Reply:
Sorry, did you mean to add an environment variable similar to RUSTC_VERSION? Is there any benefit to this?

Reply:
This is about env vars configured with `[env]` in `config.toml`.

Reply:
Yea, it probably makes sense to also set the env vars. Can you add that? It looks like `apply_env_config` might need to be `pub(crate)` or something to make that work.